src/cli/command_list.py

```diff
@@ -67,8 +68,23 @@ def applications_create(
     roles: List[str] = typer.Argument(..., help="Names of the roles to be created."),
 ) -> None:
     """
-    Create new application.
+    Create new application. e.g.: qne application create application_name role1 role2
     """
+
+    if len(roles) <= 1:
+        raise NotEnoughRoles()
+
+    invalid_chars = ['/', '\\', '*', ':', '?', '"', '<', '>', '|']
```

Comment:
Create a validation method for a name to be used in a path. Now we duplicate code (78-79 and 82-83), the exception can be the same (InvalidName).
We will need this validation again for create experiment.

Reply:
When a validation method is created it can be reused later.
```
def validate_path_name(object: str, name: str) -> None:
    """ Checks if the name of a certain object, can be used as part of a path.
    An Exception is raised when name contains an incompatible character.
    """
    invalid_chars = ['/', '\\', '*', ':', '?', '"', '<', '>', '|']
    if any(char in name for char in invalid_chars):
        raise InvalidPathName(object)
```